lms/djangoapps/onboarding/forms.py

```diff
@@ -452,76 +447,66 @@ class InterestsForm(BaseOnboardingForm):
-        self.fields['interests'] = forms.ChoiceField(
-            label=ugettext_noop('Which of these areas of organizational effectiveness are you most interested '
-                                'to learn more about? (Check all that apply.)'),
-            choices=interest_choices, widget=forms.CheckboxSelectMultiple,
-            required=False)
-
-        interested_learners_choices = get_sorted_choices_from_dict(UserExtendedProfile.INTERESTED_LEARNERS_LABELS)
-        interested_learners_choices = sorted(interested_learners_choices,
-                                             key=lambda interested_learners_choices: interested_learners_choices[0])
-        self.fields['interested_learners'] = forms.ChoiceField(
-            label=ugettext_noop('Which types of other Philanthropy University learners are interesting to you? '
-                                '(Check all that apply.)'),
-            choices=interested_learners_choices, widget=forms.CheckboxSelectMultiple,
-            required=False)
-
-        personal_goal_choices = get_sorted_choices_from_dict(UserExtendedProfile.GOALS_LABELS)
-        personal_goal_choices = sorted(personal_goal_choices,
-                                       key=lambda personal_goal_choices: personal_goal_choices[0])
-        self.fields['personal_goals'] = forms.ChoiceField(
-            label=ugettext_noop('What is your most important personal goal in joining Philanthropy University? '
-                                '(Check all that apply.)'),
-            choices=personal_goal_choices, widget=forms.CheckboxSelectMultiple,
-            required=False)
-
-        hear_about_philanthropy_choices = get_sorted_choices_from_dict(UserExtendedProfile
-                                                                       .HEAR_ABOUT_PHILANTHROPY_LABELS)
-        hear_about_philanthropy_choices = sorted(hear_about_philanthropy_choices,
-                                                 key=lambda hear_about_philanthropy_choices:
-                                                 hear_about_philanthropy_choices[0])
-        self.fields['hear_about_philanthropy'] = forms.ChoiceField(
-            label=ugettext_noop('How did you hear about Philanthropy University?'
-                                ' (Choose one. If more than one applies, please choose the source that most strongly '
-                                'influenced you to visit the website.)'),
-            choices=hear_about_philanthropy_choices, widget=forms.RadioSelect,
-            required=False)
-
-        self.fields['hear_about_philanthropy_other'] = forms.CharField(
-            max_length=255,
-            required=False,
-            widget=forms.TextInput(
-                attrs={'placeholder': ugettext_noop('Other Reason')}
-            ))
+    interests = forms.ChoiceField(
+        label=ugettext_noop('Which of these areas of organizational effectiveness are you most interested '
```

Comment:
Shouldn't we use `ugettext_lazy` here?

`ugettext_noop` only mark strings as translatable but always render string in original form i.e English here